R/followers.R

```diff
@@ -55,7 +41,7 @@ get_followers <- function(user, n = 5000,
                           page = lifecycle::deprecated()) {
   
   if (lifecycle::is_present(page)) {
-    lifecycle::deprecate_warn("1.0.0", "get_followers(page)", "get_followers(cursor)")
+    lifecycle::deprecate_warn("1.0.0", "get_followers(page = )", "get_followers(cursor = )")
```

Comment:
If the existing syntax doesn't work for you, it's a sign that we need to require lifecycle 1.0.0 in the description, and you need to update.

Reply:
Ok, I removed this change, but I'll remove the dependency to lifecycle later on in my opinion it is not worth it. I prefer to notify the user each time, specially considering th next release is a major release.

Reply:
I think that is something that’s worse discussing more - lifecycle provides a consistent way to deprecate functions and it’s designed to be a lightweight dependency.

Reply:
Maybe my earlier explanation wasn’t very good. I can fix the problem on master on Monday so you don’t need to worry about it in this PR.